pkg/helper/versions/helpers.go

```diff
@@ -26,7 +33,7 @@ func GetVersionList(r *rosa.Runtime, channelGroup string, isSTS bool, isHostedCP
 		if isSTS && !ocm.HasSTSSupport(v.RawID(), v.ChannelGroup()) {
 			continue
 		}
-		if isHostedCP {
+		if filterHostedCP {
```

Comment:
because we have 2 main use case:
- control plane versions
- node pool versions 
Both needs to an HCP minimal version filtering.
The CP versions are filtered based on `hosted_control_plane` field in each version, while nodes can have any ROSA enabled version compatible (so we don't filter if `hosted_control_plane` is enabled or not).

Hence having 2 flags: isHostedCP to apply minimal filter
filterHostedCP to filter differently based on control plan or node pool

Reply:
Thanks for the clarification.